apollo-router/src/services/execution_service.rs

```diff
@@ -141,3 +141,26 @@ impl<SF: SubgraphServiceFactory> ExecutionServiceFactory for ExecutionCreator<SF
         ExecutionRequest,
     >>::Future;
 }
+
```

Comment:
This is puzzling me. Why is this required as part of a change to the PluginTestHarness?

Reply:
This is required by a new feature of `PluginTestHarness`: it now has a `mock_execution_service` parameter, so that any stage of the pipeline can be mocked. I’ve added it to the changelog.